editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -6500,20 +6510,20 @@ void vertex() {
 	// Points are already in world space, so no need for MODEL_MATRIX anymore.
 	vec4 clip_a = PROJECTION_MATRIX * (VIEW_MATRIX * vec4(point_a, 1.0));
 	vec4 clip_b = PROJECTION_MATRIX * (VIEW_MATRIX * vec4(point_b, 1.0));
-	
+
```

Comment:
Automatic trailing space removal, can revert if desired but had difficulties changing my editor to prevent it